blackbox/docs/src/doc_tests/tests.py

```diff
@@ -445,101 +417,76 @@ def test_suite():
-               'general/ddl/storage.rst',
-               'admin/runtime-config.rst',
-               'general/ddl/show-create-table.rst',
-               'general/user-defined-functions.rst',
-               'admin/user-management.rst',
-               'admin/privileges.rst',
-               'admin/ingestion/rules.rst',
-               'general/dql/index.rst',
-               'general/dql/refresh.rst',
-               'admin/optimization.rst',
-               'general/dql/fulltext.rst',
-               'general/ddl/data-types.rst',
-               'general/occ.rst',
-               'general/information-schema.rst',
-               'general/ddl/partitioned-tables.rst',
-               'general/builtins/aggregation.rst',
-               'general/builtins/arithmetic.rst',
-               'general/builtins/scalar.rst',
-               'general/builtins/table-functions.rst',
-               'admin/system-information.rst',
-               'general/dql/selects.rst',
-               'interfaces/postgres.rst'):
-        s = docsuite('../../' + fn, parser=crash_parser,
-                     setUp=setUpLocationsAndQuotes,
-                     tearDown=tearDownLocationsAndQuotes)
+    for fn in shuffled('general/ddl/create-table.rst',
+                       'general/ddl/generated-columns.rst',
+                       'general/ddl/constraints.rst',
+                       'general/ddl/sharding.rst',
+                       'general/ddl/replication.rst',
+                       'general/ddl/column-policy.rst',
+                       'general/ddl/fulltext-indices.rst',
+                       'general/ddl/system-columns.rst',
+                       'general/ddl/alter-table.rst',
+                       'general/ddl/storage.rst',
+                       'admin/runtime-config.rst',
+                       'general/ddl/show-create-table.rst',
+                       'general/user-defined-functions.rst',
+                       'admin/user-management.rst',
+                       'admin/privileges.rst',
+                       'admin/ingestion/rules.rst',
+                       'general/dql/index.rst',
+                       'general/dql/refresh.rst',
+                       'admin/optimization.rst',
+                       'general/dql/fulltext.rst',
+                       'general/ddl/data-types.rst',
+                       'general/occ.rst',
+                       'general/information-schema.rst',
+                       'general/ddl/partitioned-tables.rst',
+                       'general/builtins/aggregation.rst',
+                       'general/builtins/arithmetic.rst',
+                       'general/builtins/scalar.rst',
+                       'general/builtins/table-functions.rst',
+                       'admin/system-information.rst',
+                       'general/dql/selects.rst',
+                       'interfaces/postgres.rst',):
+        s = docsuite(fn,
+                     setUp=setUpLocationsAndQuotes)
```

Comment:
Would be nice if we would specify the required tables in the test document itself.

Reply:
I agree, a more granular table dependency definition would be nice.
Would be another PR.